setup.cfg

```diff
@@ -31,7 +31,7 @@ universal = 1
 universal = 1
 
 [flake8]
-max-line-length = 80
+max-line-length = 88
```

Comment:
Why change this? If it's because of black I suggest adding a `pyproject.toml` with the following content:

```toml
[tool.black]
line-length = 80
python-version = ['py36']
exclude = '''
(
    __init__.py
)
'''

```

Reply:
Ah yes, indeed it's for black - I realize this is contrary to the current cookiecutter setup, sorry, I'll revert it. That exact `pyproject.toml` already exists in the repo.